pkgs/applications/networking/mailreaders/himalaya/default.nix

```diff
@@ -34,12 +36,17 @@ rustPlatform.buildRustPackage rec {
       openssl
     ];
 
-  # flag added because without end-to-end testing is ran which requires
-  # additional tooling and servers to test
+  # TODO: remove this flag once this issue is fixed:
+  # https://todo.sr.ht/~soywod/pimalaya/36
```

Comment:
Is this `pimalaya` a typo though? 🙃

Reply:
In fact `himalaya` becomes a subproject of a bigger one called `pimalaya`. The aim of this bigger project is to gather tools working the same way as `himalaya` (Rust lib + interfaces) in order to improve the way we manage our [Personal Information](https://en.wikipedia.org/wiki/Personal_information_manager). It changes nothing for `himalaya` itself, just the [website](https://pimalaya.org/), the [bug tracker](https://todo.sr.ht/~soywod/pimalaya) and the [mailing list](https://lists.sr.ht/~soywod/pimalaya) use the name `pimalaya` :slightly_smiling_face:

Reply:
I wasn't 100% sure on this one, but cool.